openslides/core/config.py

```diff
@@ -122,6 +138,7 @@ def __setitem__(self, key, value):
         try:
             db_value = ConfigStore.objects.get(key=key)
         except ConfigStore.DoesNotExist:
+            # This should never happen. All DB entries are created at startup.
```

Comment:
Or raising an own error, telling that this should never happend and **why** this should never happend, so if this error is raised, the user know, that something is messed up and may correct this.